sktime/forecasting/hf_patchtst_forecaster.py

```diff
@@ -0,0 +1,507 @@
+        set to "zeroshot", it will load the model in zero-shot forecasting
+        model with the argument `model_path` and ignore any passed `y`.
+        Note that both "finetune" and "zeroshot" mode requires a mandatory
+        passed in `model_path`.
+    patch_length : int, optional, default = 2
+        Length of each patch that will segment every univariate series.
+    context_length : int, optional, default = 4
+        Number of previous time steps used to forecast.
+    patch_stride : int, optional, default = 2
+        Length of the non-overlapping region between patches. If patch_stride
+        is less than patch_length, then there will be overlapping patches. If
+        patch_stride = patch_length, then there will be no overlapping patches.
+    random_mask_ratio : float, optional, default = 0.4
+        Masking ratio applied to mask the input data during random pretraining.
+    d_model : int, optional, default = 128
+        Dimension of the weight matrices in the transformer layers.
+    num_attention_heads : int, optional, default = 16
+        Number of attention heads for each attention layer in the transformer block
+    ffn_dim : int, optional, default = 256
+        Dimensionality of the feed forward layer in the transformer encoder.
+    head_dropout : float, optional, default = 0.2
+        Dropout probability for a head.
+    batch_size : int, optional, default = 64
+        Size of every batch during training. Reduce if you have reduced gpu power.
+    learning_rate : float, optional, default = 1e-4
+        Leaning rate that is used during training.
+    epochs : int, optional, default = 10
+        Number of epochs to use during training.
+    validation_split : float, optional, default = 0.2
+        Fraction of the data to use for validation.
+    config : dict, optional, default = {}
+        A config dict specifying parameters to initialize an untrained
+        PatchTST model. Missing parameters in the config will be automatically
+        replaced by their default values. See the PatchTSTConfig config on
+        huggingface for more details.
+    training_args : dict, optional, default = None
+        Training arguments to use for the model. If this is passed,
+        the remaining applicable training arguments will be ignored
+    compute_metrics : list or function, default = None
+        List of metrics or function to use to use during training
+    callbacks: list or function, default = None
+        List of callbacks or callback function to use during training
+    broadcasting : bool, default=False
+        if True, multiindex data input will be broadcasted to single series.
+        For each single series, one copy of this forecaster will try to
+        fit and predict on it. The broadcasting is happening inside automatically,
+        from the outerside api perspective, the input and output are the same,
+        only one multiindex output from ``predict``.
```

Comment:
this topics has been in discussion, but do we need to keep this option with new models?

Reply:
Honestly, not too sure, it can be removed or included either way.. Do you perhaps have a link to the issue in which this discussion occurs?